### PR TITLE
[FastPR][Fluid] Monolithic solver inconsistency

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
In this PR I'm (finally) renaming the `navier_stokes_solver_vmsmonolithic.py` to `navier_stokes_monolithic_solver.py`. The  rationale is
1. This solver implements more than VMS formulations (all of them monolithic).
2. Following the application naming convention (note that this name is inherited from the one we used to have in the pre-analysis stage era).
To avoid breaking backwards compatibility, I'm creating an auxiliary module that throws a deprecation warning and calls the constructor of the new one. This will eventually be removed in the future.

### CODE REVIEW

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_monolithic_solver.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import base class file
+from KratosMultiphysics.FluidDynamicsApplication.fluid_solver import FluidSolver
+
+class StabilizedFormulation(object):
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/python_scripts/navier_stokes_monolithic_solver.py]:
```suggestion
class StabilizedFormulation:
```

File: applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
     # Solvers for MPI parallelism
     elif (parallelism == "MPI"):
         if solver_type == "monolithic" or solver_type == "Monolithic":
-            solver_module_name = "trilinos_navier_stokes_solver_vmsmonolithic"
+            solver_module_name = "trilinos_navier_stokes_monolithic_solver"
 
         elif solver_type == "fractional_step" or solver_type == "FractionalStep":
             solver_module_name = "trilinos_navier_stokes_solver_fractionalstep"
```
Comment (sunethwarna) [inline applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py:49]: We have some inconsistent naming still... Are we gonna change those also?

**Review by sunethwarna — DISMISSED** [review]
Nice, minor comment only :).

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py
Comment (philbucher) [inline applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py:1]: you could keep this file with a dummy import of the solver to keep backward compatibility

File: applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py:49]: Yes. But I'd do so in a separate PR.

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py:1]: mmm... Isn't this what is done with current behavior?

**Comment by rubenzorrilla** [comment]
@jgonzalezusua FYI.

**Comment by jgonzalezusua** [comment]
> @jgonzalezusua FYI.
thanks for the information

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py
Comment (philbucher) [reply applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py:1]: you are right, it seemed like the file was deleted. Nice job :+1:

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**